tests/screenshot/apps/components/TabLayout.js

```diff
@@ -90,13 +105,97 @@ const TabLayoutTests = [
 		component: <TabLayout collapsed>{tabsWithIconsDisabled}</TabLayout>,
 		wrapper: {full: true}
 	},
+	{
+		component: <TabLayout collapsed selected={0}>{tabsWithIconsDisabled}</TabLayout>,
+		wrapper: {full: true},
+		focus: true
+	},
 	{
 		component: <TabLayout orientation="horizontal">{tabs}</TabLayout>,
 		wrapper: {full: true}
 	},
 	{
 		component: <TabLayout orientation="horizontal" selected={3}>{tabs}</TabLayout>,
 		wrapper: {full: true}
+	},
+	{
+		component: <TabLayout anchorTo="left">{tabs}</TabLayout>,
+		wrapper: {full: true}
+	},
+	{
+		component: <TabLayout anchorTo="right">{tabs}</TabLayout>,
+		wrapper: {full: true}
+	},
+	{
+		component: <TabLayout anchorTo="start">{tabs}</TabLayout>,
+		wrapper: {full: true}
+	},
+	{
+		component: <TabLayout anchorTo="end">{tabs}</TabLayout>,
+		wrapper: {full: true}
+	},
+	{
+		component: <TabLayout
+			dimensions={{
+				tabs: {
+					collapsed: 300,
+					normal: 900
```

Comment:
Result of this test title is overlapped with focused tap. Is this the intended test?

Reply:
It was more intended to check for different normal values to catch if it displays differently. (the same for the bellow one)